lib/v1.mapper.js

```diff
@@ -0,0 +1,343 @@
+/**
```

Comment:
usually we don't want to suffix js files with `.mapper` or something like that. Some people have `index.es6.js` in that it tells them extra information about what kind of javascript they're writing. In this case, mapper is just the functionality/logic of the file and we should avoid using those kinds of suffixes imo.